lib/jsonwp-proxy/proxy.js

```diff
@@ -27,20 +30,32 @@ class JWProxy {
       reqBasePath: DEFAULT_BASE_PATH,
       sessionId: null,
       timeout: DEFAULT_REQUEST_TIMEOUT,
-      keepAlive: false,
+      keepAlive: true,
```

Comment:
why change this value?

Reply:
In general it is better to have keepAlive enabled by default, since it saves time while sending requests to the downstream server. It was previously enabled, but we've disabled it, because of some internal node + system buffers issues. It was long time ago, so I'd expect correct node versions work much better now and should handle buffering properly.

I'll spend some time debugging the server close helper to understand why keepAlive is blocking it

Reply:
Actually here is the thing: https://github.com/nodejs/node-v0.x-archive/issues/9066